en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md

```diff
@@ -0,0 +1,215 @@
+# Configuring Keystores in API-M Analytics
+
+WSO2 products use asymmetric cryptography by default for authentication and data encryption. In asymmetric cryptography, keystores (with key pairs and certificates) are created and stored for the product. Keystore is a repository where private keys and certificates can be stored. It is possible to have multiple keystores so that the keys used for different use cases are kept unique. For more information about keystores and its concepts, see [About Asymmetric Cryptography]({{base_path}}/install-and-setup/setup/security/configuring-keystores/keystore-basics/about-asymetric-cryptography).
+
+In WSO2 API-M Analytics a set of keystores can be configured and used for different use cases in both dashboard and worker profiles.
+
+The `wso2carbon.jks` keystore file, which is shipped with all WSO2 products, is used as the default keystore for all functions. However, in a production environment, it is recommended to [create new keystores]({{base_path}}/install-and-setup/setup/security/configuring-keystores/keystore-basics/creating-new-keystores) with new keys and certificates. If you have created a new keystore and updated the `client-truststore.jks` file, which resides in the `<API-M_ANALYTICS_HOME>/resources/security` directory, you must update the `deployment.yaml` file of the corresponding profile in order to make the keystore work.
+
+## Configuring keystores in dashboard profile
+
+!!! note
+    Other than explicitly mentioned, you need to do all the following configuration updates related to the dashboard profile in the `<API-M_ANALYTICS_HOME>/conf/dashboard/deployment.yaml` file.
+
+1. Update the Databridge configurations.
+
+    This is used for authenticating communication over SSL/TLS between API Manager and API-M Analytics. Handles thrift and binary transports.  
+
+    ```yaml
+    databridge.config:
+      ... 
+      keyStoreLocation : ${sys:carbon.home}/resources/security/wso2carbon.jks
+      keyStorePassword : wso2carbon
+      ... 
+    ```
+    The elements in the above configuration are described below:
+       
+    | Element                | Description                                                      |
+    |------------------------|------------------------------------------------------------------|
+    | **keyStoreLocation**   | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
```

Comment:
should we mention about the formats supported as well?

Reply:
@fazlan-nazeem , please share your views.

Reply:
do you mean other keystore file formats?

Reply:
@shaniR , please clarify @fazlan-nazeem 's Q

Reply:
we support .jks only.

Reply:
fixed